### PR TITLE
Add landing page header

### PR DESCRIPTION
**Pull request by omarkmu**
Added a header to the landing page, and fixed a bug that stopped the landing page from displaying.

The header's styling can (and should) be improved if we can add bootstrap.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  return (
+    <div className="navbar">
+      <div className="nav-left">
+        <Link to="/" className="nav-link">
+          <BotIcon full />
+        </Link>
+      </div>
+      <div className="nav-right">
+        <AnchorButton to="/login" text="Login" />
+        <AnchorButton to="/registration" text="Register" />
```
Comment (BrandonDwyer) [inline src/components/Header.js:40]: Good job adding buttons. Should we remove the redirect links from the landing page and only have a login and register buttons in the Header only?

**Comment by BrandonDwyer** [comment]
The comment I left is a design decision left up to you to decide. I think having both will be optimal but just asking if you think one might be better than having just one redirect in the header.

File: src/components/Header.js (hunk 1 above)
Comment (omarkmu) [reply src/components/Header.js:40]: Good call, I'll go ahead and remove them.

File: src/pages/Landing.js [hunk 2]
```diff
@@ -29,8 +29,6 @@ export default function Landing() {
         shortening ailments that could hamper our quality of life.
       </p>
       <h3>Get started today on your new life today!</h3>
```
Comment (BrandonDwyer) [inline src/pages/Landing.js]: If you are deleting these hyperlinks should we remove this header?

Comment (omarkmu) [reply src/pages/Landing.js]: I don't think the call to action necessarily needs to be removed (there isn't much content on the landing page, so anything helps), but I can go ahead and take that out too.

Comment (BrandonDwyer) [reply src/pages/Landing.js]: The landing page content is pretty bare-bones for what is required there in the syllabus.

**Review by BrandonDwyer — APPROVED** [review]

**Pull request merged**